adam/build/Dockerfile

```diff
@@ -1,19 +1,7 @@
-FROM ubuntu
+FROM computationalgenomicslab/spark-and-maven:1.5.2.3.3.3-build
```

Comment:
Yeah, the name is lame, but I was really expecting someone to get hyped because of the version string.